resources/serverless/values.yaml

```
@@ -176,8 +176,8 @@ containers:
   manager:
     resources:
       limits:
-        cpu: 500m
-        memory: 160Mi
+        cpu: 600m
```

I would even accommodate bigger CPU spikes for serverless-ctrl-mngr

```suggestion
        cpu: 1500m
```